### DISCUSSION TITLE
windows and brave search

### DISCUSSION THREAD

**Discussion by scottsuhy** [modelcontextprotocol/.github#59] [General]
### Pre-submission Checklist

- [X] I have verified that this discussion would not be more appropriate as an issue in a specific repository
- [X] I have searched existing discussions to avoid duplicates

### Discussion Topic

on the Windows instructions, you will note that the instructions have you use "npx" but you need to replace it with "node" and change the arguments to the entire path

use : "npm root -g" to get the node path and add it to the beginning of the string, then add "/dist/index.js" to the end of the string
```
{
    "mcpServers": {
        "brave_search": {
            "command": "node",
            "args": ["C:/Users/scott/AppData/Roaming/npm/node_modules/@modelcontextprotocol/server-brave-search/dist/index.js"],
            "env": {
                "BRAVE_API_KEY": "YOUR API KEY"
            }
        },
        "sqlite": {
            "command": "uvx",
            "args": [
                "mcp-server-sqlite",
                "--db-path",
                "C:\\Users\\scott\\PhpstormProjects\\boom\\python\\roi_database6.db"
            ]
        }
    }
}
```
this video was helpful to figure it out https://www.youtube.com/watch?v=xcj_5xl1RNs

**Comment by jspahrsummers** [comment]
The use of `npx` on Windows [should be fixed](https://github.com/modelcontextprotocol/servers/issues/40#issuecomment-2511901524) now.
